packages/accounts-controller/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@metamask/accounts-controller",
+  "version": "1.0.0",
+  "description": "Manages internal accounts",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/accounts-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build:docs": "typedoc",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/accounts-controller",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@metamask/base-controller": "^3.2.1",
+    "@metamask/eth-snap-keyring": "^0.2.2",
+    "@metamask/keyring-api": "^0.2.5",
+    "@metamask/keyring-controller": "^7.4.0",
+    "@metamask/snaps-controllers": "^1.0.1",
+    "@metamask/snaps-utils": "^1.0.1",
+    "@metamask/utils": "^6.2.0",
+    "eth-rpc-errors": "^4.0.2",
+    "ethereumjs-util": "^7.0.10",
+    "immer": "^9.0.6",
+    "nanoid": "^3.1.31",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@metamask/auto-changelog": "^3.1.0",
+    "@types/jest": "^27.4.1",
+    "@types/readable-stream": "^2.3.0",
+    "deepmerge": "^4.2.2",
+    "jest": "^27.5.1",
+    "sinon": "^9.2.4",
```

Comment:
I think this can be removed

Reply:
We still need `@types/readable-stream`, and `jest` but the rest can be removed. Changed in 2c2fa52656e82694c8b4e48440e718cb8d3fd86d

Reply:
The comment was referring to `sinon`, but good that we can remove `deepmerge` too :smile: